index.html

```diff
@@ -16,13 +16,13 @@
 </header>
     <main>
         <h1>Welcome to Jordan Poulin's CSD120 Website!</h1>
-            <p>
-             I'm currently a first year Computer Programmer student at Sault College in Sault Ste. Marie, Ontario. The majority of <br>
-             my IT experience is in hardware. I'm an avid gamer and enjoy building PCs and figuring out how computer components work. <br>
```

Comment:
Unless you really need a line to wrap (as we did in the Haiku below), you should avoid using <br> to manage whitespace.  Later in the term we will learn ways to manage the width of `<p>` elements using CSS.